src/System.Windows.Forms/tests/AccessibilityTests/DataBindingExample.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Windows.Forms;
+
+namespace AccessibilityTests
+{
+    public partial class DataBindingExample : Form
+    {
+        private readonly List<Student> _studentA = new List<Student>();
+        private readonly List<Student> _studentB = new List<Student>();
+
+        public DataBindingExample()
+        {
+            InitializeComponent();
+        }
+
+        private void Binding_Data_Load(object sender, EventArgs e)
+        {
+            for (int i = 0; i < 6; i++)
+            {
+                _studentA.Add(new Student(i, "Name 1" + i, "Male"));
+                _studentB.Add(new Student(i * 2, "Name 11" + i * 2, "Female"));
+            }
+
+            // Binding Data For ListBox & ComboBox & CheckedListBox controls by using DataSource property
+            listBox1.DataSource = _studentA;
+            comboBox1.DataSource = _studentB;
+
+            listBox1.DisplayMember = "StudentName";
```

Comment:
To minimise maintenance overhead when properties are renamed, it is better to use [`nameof()` expression](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/nameof):
```suggestion
            listBox1.DisplayMember = nameof(Student.StudentName);
```

The same applies below.